pkgs/test/by-name/check-references/README.md

```diff
@@ -0,0 +1,3 @@
+Usage: nix-reference-check [FILES]
```

Comment:
```suggestion
Usage: check-references [DIR]
```

At least when I built it, the binary that got built was called `check-references`.

Also, it insists on being passed a directory; you can't pass it just a single `.nix` file.

Reply:
This is outdated, the check is now in https://github.com/NixOS/nixpkgs/pull/250885, called `nixpkgs-check-by-name <NIXPKGS>`, and there's no support for files (it takes a Nixpkgs path now).